modules/oe_theme_content_consultation/oe_theme_content_consultation.module

```diff
@@ -0,0 +1,106 @@
+<?php
+
+/**
+ * @file
+ * Module file used for theming the Consultation content type.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Entity\Display\EntityViewDisplayInterface;
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Url;
+use Drupal\oe_content\CallEntityWrapperInterface;
+use Drupal\oe_content_consultation\ConsultationNodeWrapper;
+
+/**
+ * Implements hook_theme().
+ */
+function oe_theme_content_consultation_theme() {
+  return [
+    'oe_theme_content_consultation_closed_text' => [
+      'variables' => [
+        'text' => '',
+      ],
+    ],
+  ];
+}
+
+/**
+ * Implements hook_ENTITY_TYPE_view_alter().
+ */
+function oe_theme_content_consultation_node_view_alter(array &$build, EntityInterface $entity, EntityViewDisplayInterface $display) {
+  if ($entity->bundle() !== 'oe_consultation') {
+    return;
+  }
+
+  if ($build['#view_mode'] === 'full') {
+    // Override field labels.
+    if ($entity->get('oe_departments')->count() === 1) {
+      $build['oe_departments']['#title'] = t('Department');
+    }
+
+    // Show respond to the consultation information based status.
+    $consultation_node_wrapper = ConsultationNodeWrapper::getInstance($entity);
+    $status = $consultation_node_wrapper->getStatus();
+    $build['oe_consultation_response_button']['#access'] = FALSE;
+    if ($status === CallEntityWrapperInterface::STATUS_OPEN) {
+      $build['extra_field_oe_consultation_closed_status_text']['#access'] = FALSE;
+      if (!$entity->get('oe_consultation_response_button')->isEmpty()) {
+        $build['oe_consultation_response_button']['#access'] = TRUE;
+      }
+    }
+    if ($status === CallEntityWrapperInterface::STATUS_CLOSED) {
+      $build['oe_consultation_guidelines']['#access'] = FALSE;
+    }
+  }
+}
+
+/**
+ * Prepares variables for in-page navigation.
+ *
+ * Default template: oe-theme-helper-in-page-navigation.html.twig.
+ *
+ * @param array $variables
+ *   An array of variables.
+ */
+function oe_theme_content_consultation_preprocess_oe_theme_helper_in_page_navigation__node__oe_consultation(array &$variables): void {
+  if ($variables['entity']->get('oe_consultation_contacts')->count() > 1) {
+    foreach ($variables['items'] as $index => $item) {
+      if ($item['label'] === (string) t('Contact')) {
+        $variables['items'][$index]['label'] = t('Contacts');
+      }
+    }
+  }
+}
+
+/**
+ * Implements hook_preprocess_field().
+ *
+ * Add formatting to Consultation text fields.
+ */
+function oe_theme_content_consultation_preprocess_field(&$variables, $hook): void {
+  $text_formatted_fields = [
+    'oe_consultation_additional_info',
+    'oe_consultation_outcome',
+    'oe_consultation_legal_info',
+    'oe_consultation_guidelines',
+    'oe_consultation_target_audience',
+    'oe_consultation_aim',
+  ];
+  if (in_array($variables['field_name'], $text_formatted_fields)) {
+    $variables['attributes']['class'][] = 'ecl-editor';
```

Comment:
How did we do this on other content types? I think we used templates right? If so let's stick with that method or improve it, but better to keep the general approach to this consistent.